nipoppy/workflows/pipeline.py

```diff
@@ -746,11 +746,12 @@ def _submit_hpc_job(self, participants_sessions):
     def run_cleanup(self):
         """Log a summary message."""
         if self.n_total == 0:
-            self.logger.warning(
+            self.logger.error(
```

Comment:
🍒 I prefer a warning here. If you have a strong preference for error, then I'm fine with it.